### ISSUE TITLE
Is there a way to add the accessibleName for the <ui5-form-group> component ?

### ISSUE THREAD

**Issue by Lukas742** [SAP/ui5-webcomponents-react#7663]
### Discussed in https://github.com/SAP/ui5-webcomponents-react/discussions/7662

<div type='discussions-op-text'>

<sup>Originally posted by **VidhiNPatel** August 19, 2025</sup>
Currently, Axe Devtool is reporting a missing aria-label on the <ui5-form-group> where role="form" is specified. According to accessibility standards, an aria-label should be present when a role is defined.

<img width="1698" height="744" alt="Screenshot 2025-08-19 at 2 43 52 PM" src="https://github.com/user-attachments/assets/f47d5721-feca-4e92-bbd9-fe8d25399c7d" />
<img width="1358" height="745" alt="Screenshot 2025-08-19 at 2 37 06 PM" src="https://github.com/user-attachments/assets/7efa739d-4fcd-42d8-a9c8-075c9de142e3" />
</div>
